### ISSUE TITLE
Proposition de plan - rester fonctionnel

### ISSUE THREAD

**Issue by artragis** [artragis/tutoriels-et-articles-les-nouveautes-de-la-version-159#6]
# Il était une fois deux modules si proches mais si différents en même temps
- avant on a tuto + article <- faire du story telling?
- du coup les articles ça prend du retard fonctionnel et ça a des bugs 
- maintenant les articles ont tout, notamment la version béta, l'intégration à la zep-03, la gallerie...
  Demander à MaxRoyo une Clem 2.0?
# Dans un monde où seuls les petits et les grands existent...
- assouplissement de la structure
- premier effet : arrivée des moyens tuto
- arrivée du déplacement avancé
- rappeler la règle de création des sections et conteneurs
- les articles ont des extraits = meilleurs SEO + plus facile à rédiger
# où rédiger devient plus facile
- améliorations cosmétiques : ordre des catégorie, interface de diff...
- les éditeurs externes? importation du markdown ET DES IMAGES (yeeepi)

autres?
# je n'ai pas fini :

(photo du grand père dans le DA jacky chan)
- la doc améliorée
- la stabilité testée un max
- les améliorations de perfs (cf #5 )

autre?

**Comment by GerardPaligot** [comment]
> avant on a tuto + article <- faire du story telling?

Ce que tu voudrais mettre en avant, c'est le côté distinct des 2 types de contenu ? Si oui, c'est une bonne idée. Reste plus qu'à savoir quoi raconter autour.

> du coup les articles ça prend du retard fonctionnel et ça a des bugs

En fait, c'est partiellement vrai. Effectivement, les articles étaient dans un module distinct mais le problème repose surtout sur les similitudes entre ces 2 modules. Un bug corrigé dans le module des tutos pouvait être présent dans celui des articles mais pas corrigé **et inversement**.

Du coup, à la place d'avoir un module sur 2 qui se détériorait, on avait 2 modules qui se détérioraient.

> maintenant les articles ont tout, notamment la version béta, l'intégration à la zep-03, la gallerie... Demander à MaxRoyo une Clem 2.0?

Maintenant les articles sont techniquement des tutoriels mais fonctionnellement des articles. Ca veut dire qu'un bug corrigé sur les articles aura un impact direct sur les tutoriels. C'est à la fois bien et pas bien (je pense que c'est important d'expliquer les risques aussi).

Dorénavant, quand un correctif sera proposé pour les articles, il faudra toujours garder à l'esprit des impacts sur les autres contenus. Aujourd'hui, nous avons que des tutoriels mais demain ?

> Dans un monde où seuls les petits et les grands existent...

Comme je l'explique dans [cette issue](https://github.com/artragis/tutoriels-et-articles-les-nouveautes-de-la-version-159/issues/3), je verrais bien une mise en avant de certaines fonctionnalités comme des killers features.

> rappeler la règle de création des sections et conteneurs

Dans cet article, nous présentons le nouveau module. Est-ce le lieu pour y placer de la "documentation" d'utilisation ?

> où rédiger devient plus facile

Je ne comprends pas trop la différence avec la section précédente. Ce sont aussi des améliorations.

> je n'ai pas fini :

Ca rejoint un peu [cette issue](https://github.com/artragis/tutoriels-et-articles-les-nouveautes-de-la-version-159/issues/5). Hormis le fait que je ne vois pas trop ce que la documentation vient faire là. Pour le reste, je pense que c'est une bonne chose d'insister sur la stabilité du nouveau module et des meilleures stats.

**Comment by pierre-24** [comment]
> Comme je l'explique dans cette issue, je verrais bien une mise en avant de certaines fonctionnalités comme des killers features.

Pour. Sans ce jeter des fleurs, autant montrer que c'est pas juste une refacto de code pour passer en CBV :)

> Dans cet article, nous présentons le nouveau module. Est-ce le lieu pour y placer de la "documentation" d'utilisation ?

Pour moi, il en faut un minimum, au moins pour indiquer aux auteurs que extrait → section. Après, il suffit de renvoyer à la page de doc de RTD pour la suite :)

> Pour le reste, je pense que c'est une bonne chose d'insister sur la stabilité du nouveau module et des meilleures stats.

+1, pareil qu'au dessus, il faut montrer que c'est pas juste de la masturbation intellectuelle :)

**Comment by Situphen** [comment]
> c'est pas juste de la masturbation intellectuelle :)

Ah bon ? :D

**Comment by artragis** [comment]
pour la doc je voyais plutôt ça sur l'angle : maintenant vous pouvez vraiment faire un logiciel de rédaction car tout est absolument documenté, notamment les manifest et comment ils évolueront.

**Comment by pierre-24** [comment]
On peut le mentionner. Mais n'oublie pas que le public, c'est avant tout des auteurs plus que des codeurs ;)

**Comment by artragis** [comment]
yep, juste mentionner (et puis j'avoue c'est aussi un tacle à firm1 car 
reproduire les anciennes features non documentées c'était relou)

**Comment by artragis** [comment]
même qu'on a déjà la clem 2.0 par blackline 

![clem 2.0](https://zestedesavoir.com/media/galleries/877/6943c548-1a50-45e4-85de-e3967a0e4444.png.960x960_q85.png)

**Comment by pierre-24** [comment]
awesome <3
